crates/jit/src/code_memory.rs

```diff
@@ -312,7 +312,7 @@ impl CodeMemory {
             }
         }
 
-        // Register all unwind entiries for functions and trampolines.
+        // Register all unwind entires for functions and trampolines.
```

Comment:
*entries

Reply:
Are you sure I didn't mean uncastrated male horse? 😱 

Good eye! I selected the wrong word in the spell checker drop-down menu it seems.